core/module/instance_methods_spec.rb

```diff
@@ -6,6 +6,11 @@
     methods = ModuleSpecs::Parent.instance_methods(false)
     methods.should_not include(:undefed_method)
   end
+  
+  it "does not consider methods as superclass methods if invoked on the parent module" do
```

Comment:
I do not understand the description. Is it about instance_methods only returning the methods of the module?
But that module does not have a parent, right?